src/app/wizard/intro/intro.component.html

```diff
@@ -5,28 +5,22 @@
   <h1>
     Welcome to the Launchpad
   </h1>
+  <h2>You are moments away from continuous delivery of a new <br>
+    application, built and deployed on the <br />
+    OpenShift cloud platform.</h2>
+  <h3>Currently supporting runtimes:</h3>
   <div class="lead">
     <p>
-      You are moments away from continuous delivery of a new application,
-      built and deployed on the OpenShift cloud platform.
-
-      Here we'll help you:
-    </p>
-    <ul>
-      <li>Choose your Mission, an initial goal like "open up an HTTP endpoint" or "interact with a database".</li>
-      <li>Select a Runtime, one of <a href="http://vertx.io/" target="eclipsevertx">Eclipse Vert.x</a> for asynchronous and
-        reactive development, <a href="http://wildfly-swarm.io/" target="wildflyswarm">WildFly Swarm</a> for Microservices with Java EE,
-        or <a href="https://projects.spring.io/spring-boot/" target="springboot">Spring Boot</a> for opinionated, standalone Spring applications.</li>
-      <li>Access your Booster, a fully-functioning starter application which will be live and available in minutes.</li>
-    </ul>
-    <p>
-      Along the way we'll (with your permission!) push the Booster into your GitHub namespace, set up a build pipeline for
-      continuous delivery, and ensure it's triggered to run on each push to your git remote.
-    </p>
-    <p>
-      Let's get you into orbit.
+      Images here: Eclipse Vert.x | WildFly Swarm | Spring Boot
     </p>
   </div>
+  <h4></h4>
+  <div class="blank-slate-pf-main-action">
+    <button class="btn btn-primary btn-lg btn-reg" (click)="launch()">
```

Comment:
I guess we don't want to start the wizard on this button, but got to "https://manage.openshift.com/openshiftio" so better would be to make it a link:
```html
<a href="https://manage.openshift.com/openshiftio" class="btn btn-primary btn-lg btn-reg">
```